tools/helper.psm1

```diff
@@ -89,24 +89,21 @@ function New-gRPCAutoGenCode
         [switch] $Force
     )
 
-    if ($Force -or -not (Test-Path "$RepoRoot/src/Messaging/FunctionRpc.cs") -or
-                   -not (Test-Path "$RepoRoot/src/Messaging/FunctionRpcGrpc.cs"))
+    if ($Force -or -not (Test-Path "$RepoRoot/src/Messaging/gRPC/FunctionRpc.cs") -or
```

Comment:
Should we add "$RepoRoot/src/Messaging/gRPC/NullableString.cs" and "$RepoRoot/src/Messaging/gRPC/ClaimsIdentityRpc.cs" ?

Reply:
I'd say no. This test is more of a hint to see if we can skip this in our local build. It's like the resource strings and `Start-ResGen`, you just have to run the command when the protobuff file is updated locally.
Otherwise, we have to change helper.psm1 very time there is a change to the protobuff file.